api/src/main/resources/liquibase.xml

```diff
@@ -712,4 +712,22 @@
             ALTER TABLE patient_appointment MODIFY end_date_time datetime default null;
         </sql>
     </changeSet>
+    <changeSet id="add-patient-appointment-encounter-map-table-20240429" author="mgoodrich">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="patient_appointment_encounter_map" />
```

Comment:
What about naming this `patient_appointment_fulfillment_encounter`?

Reply:
The OpenMRS convention seems to be end these tables with "map" (but, yes, if we change the name of the parameter to fulfillment or fulfilling, we should add that to this table.